netmanager/src/views/layouts/common/Topbar.js

```diff
@@ -273,7 +275,7 @@ const Topbar = (props) => {
         </Hidden>
 
         <div className={classes.flexGrow} />
-        {isAuthenticatedUser ? (
```

Comment:
Hi @Codebmk , please shed more light on this change.
Why just check if the user object is NOT empty instead of the authentication status of a user?

Reply:
When the user is logged in, the user object returns the user details while when the user is logged out, the user object should be empty. Just like if I wanted to display a list of sites, I would check if the sites list is empty or not to determine whether to display it.

Reply:
Thanks @Codebmk , clear now.